src/Form/Control/DropdownCascade.php

```diff
@@ -35,46 +35,61 @@ protected function init(): void
         }
 
-        $this->cascadeControlValue = $_POST[$this->cascadeControl->name] ?? $this->cascadeControl->field->get();
+        $this->cascadeFromControlValue = $_POST[$this->cascadeFromControl->name] ?? $this->cascadeFromControl->field->get();
 
-        $this->model = $this->cascadeControl->model ? $this->cascadeControl->model->ref($this->reference) : null;
+        $this->model = $this->cascadeFromControl->model ? $this->cascadeFromControl->model->ref($this->reference) : null;
 
-        // setup initial values and add it via dropdownOptions.
-        $values = $this->getJsValues($this->getNewValues((string) $this->cascadeControlValue), (string) $this->field->get());
+        // populate default dropdown values and add it via dropdownOptions.
+        $values = $this->getJsValues($this->getNewValues($this->cascadeFromControlValue), $this->field->get());
         $this->dropdownOptions = array_merge($this->dropdownOptions, ['values' => $values]);
 
         // js to execute for the onChange handler of the parent dropdown.
         $expr = [
             function ($t) {
                 return [
-                    $this->js()->dropdown('change values', $this->getNewValues((string) $this->cascadeControlValue)),
+                    $this->js()->dropdown('change values', $this->getNewValues($this->cascadeFromControlValue)),
                     $this->js()->removeClass('loading'),
                 ];
             },
             $this->js()->dropdown('clear'),
             $this->js()->addClass('loading'),
         ];
 
-        $this->cascadeControl->onChange($expr, ['args' => [$this->cascadeControl->name => $this->cascadeControl->jsInput()->val()]]);
+        $this->cascadeFromControl->onChange($expr, ['args' => [$this->cascadeFromControl->name => $this->cascadeFromControl->jsInput()->val()]]);
+    }
+
+    /**
+     * Allow initializing CascadeDropdown with preset value.
+     *
+     * @param string|int $initialValue       The initial ID value to set this dropdown using reference model values
+     * @param string|int $fromReferenceValue The Cascade from reference ID value where reference model values
+     */
+    public function setInitialValues($initialValue, $fromReferenceValue): self
+    {
+        $this->dropdownOptions['values'] = $this->getJsValues($this->getNewValues($fromReferenceValue), $initialValue);
+
+        return $this;
     }
 
     /**
      * Generate new dropdown values based on cascadeInput model selected id.
      * Return an empty value set if id is null.
+     *
+     * @param string|int $id
      */
-    public function getNewValues(string $id): array
+    public function getNewValues($id): array
     {
         if (!$id) {
             return [['value' => '', 'text' => $this->empty, 'name' => $this->empty]];
         }
 
-        $model = $this->cascadeControl->model->load($id)->ref($this->reference);
+        $model = $this->cascadeFromControl->model->tryLoad($id)->ref($this->reference);
```

Comment:
why `tryLoad`?

Reply:
No need to throw an exception - showing an empty dropdown is enough if the model is not loaded.